### PR TITLE
docs: Document theme and component layer variables #289

### LINKED ISSUE #289: Document theme and component layer variables

### PR DESCRIPTION
**Pull request by szuperaz**
closes #289

### CODE REVIEW

File: docusaurus/docs/Angular/concepts/customization.mdx [hunk 1]
```diff
@@ -3,7 +3,7 @@ id: customization
 title: Customization
 ---
 
-Apart from customizing the [themeing and CSS](./themeing-and-css-v2.mdx) of UI components it's also possible to provide your own template to completely override parts of the UI to further improve the flexibility of our SDK. This guide shows you the different custom templates you can provide.
+Apart from customizing the [themeing and CSS](../theming/introduction.mdx) of UI components it's also possible to provide your own template to completely override parts of the UI to further improve the flexibility of our SDK. This guide shows you the different custom templates you can provide.
```
Comment (arnautov-anton) [inline docusaurus/docs/Angular/concepts/customization.mdx:6]:
```suggestion
Apart from customizing the [theming and CSS](../theming/introduction.mdx) of UI components it's also possible to provide your own template to completely override parts of the UI to further improve the flexibility of our SDK. This guide shows you the different custom templates you can provide.
```

File: docusaurus/docs/Angular/concepts/themeing-and-css.mdx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 id: themeing
 sidebar_position: 1
-title: Themeing and CSS
+title: Themeing and CSS (deprecated)
```
Comment (arnautov-anton) [inline docusaurus/docs/Angular/concepts/themeing-and-css.mdx:4]:
```suggestion
title: Theming and CSS (deprecated)
```

File: docusaurus/docs/Angular/theming/introduction.mdx [hunk 3]
```diff
@@ -18,7 +19,7 @@ The default chat UI has a new theming and customization system, this page contai
 - Better customization
 - Support for RTL layout
 
-The [old theme](./themeing-and-css.mdx) can still be used with the latest SDK versions, but it's now deprecated and will be removed in a future release.
+The [old theme](../concepts/themeing-and-css.mdx) can still be used with the latest SDK versions, but it's now deprecated and will be removed in a future release.
```
Comment (arnautov-anton) [inline docusaurus/docs/Angular/theming/introduction.mdx:22]:
```suggestion
The [old theme](../concepts/theming-and-css.mdx) can still be used with the latest SDK versions, but it's now deprecated and will be removed in a future release.
```
We'll need to change the name of the file too. :)

File: docusaurus/docs/Angular/concepts/customization.mdx (hunk 1 above)
Comment (szuperaz) [reply docusaurus/docs/Angular/concepts/customization.mdx]: Updated it, thanks

File: docusaurus/docs/Angular/theming/introduction.mdx (hunk 3 above)
Comment (szuperaz) [reply docusaurus/docs/Angular/theming/introduction.mdx]: Done

File: docusaurus/docs/Angular/concepts/themeing-and-css.mdx (hunk 2 above)
Comment (szuperaz) [reply docusaurus/docs/Angular/concepts/themeing-and-css.mdx]: Done

**Review by arnautov-anton — APPROVED** [review]

**Pull request merged**